### PR TITLE
Support aws_session_token param in aws credentials file

### PR DESCRIPTION
**Pull request by yaussat**
Currently, `GetAwsCredentials` method reads only the first three lines of the provided credentials file, ignoring the `aws_session_token` parameter. This PR fixes this logic, making sure that we read `aws_session_token`, if it is present in the provided credentials file.

### CODE REVIEW

File: cc/integration/awskms/aws_kms_client.cc [hunk 1]
```diff
@@ -90,7 +90,7 @@ StatusOr<std::string> Read(const std::string& filename) {
 // Extracts a value of 'name' from 'line', where 'line' must be in format:
 // name = some_value
 StatusOr<std::string> GetValue(absl::string_view name, absl::string_view line) {
-  std::vector<std::string> parts = absl::StrSplit(line, '=');
+  std::vector<std::string> parts = absl::StrSplit(line, " = ");
```
Comment (bkotsopoulossc) [inline cc/integration/awskms/aws_kms_client.cc:93]: Why was this change needed? Seems it already strips whitespace below

Comment (yaussat) [reply cc/integration/awskms/aws_kms_client.cc:93]: `aws_session_token` can sometimes contain "=", and therefore, when splitting the line by "=", we can end up with more than two chunks. Splitting by " = " ensures that we always have two chunks per line.
